ctapipe/io/containers.py

```diff
@@ -75,6 +75,10 @@ class RawCameraContainer(Container):
     """
     Storage of raw data from a single telescope
     """
+    event_number = Item(-1, "telescope event number")
+    num_channels = Item(-1, "number of channels per pixel")
```

Comment:
Also, it's true that the R0 formats may be quite different per telescope, only DL0 should be common.